lib/mime.ex

```diff
@@ -164,6 +164,16 @@ defmodule MIME do
     unquote(Macro.escape(custom_types))
   end
 
+  @doc """
+  Returns a mapping of all known types to their extensions,
+  including custom types compiled into the MIME module.
```

Comment:
Might help visual learners 😉 

```suggestion
  including custom types compiled into the MIME module.
  
  ## Examples
  
      known_types()
      #=> %{"application/json" => ["json"], ...}

```